plugins/source/aws/README.md

```diff
@@ -73,9 +73,10 @@ export AWS_PROFILE=myprofile
 or, configure your desired profile in the `local_profile` field:
 
 ```yaml title="aws.yml"
-accounts:
-  id: <account_alias>
-  local_profile: myprofile
+...
```

Comment:
I think we should have a convention in our docs where we don't indent and plugin documentation are always assumed it will be under the `spec`, otherwise it just adds boilerplate code and `...` will be in every snippet

Reply:
Hmm, I think someone is going to be confused if we don't indent (or somehow indicate that it's a partial snippet). But maybe we can go with that convention until the first user comes to Discord asking why their unindented config doesn't work, then talk about it again 😃

Reply:
Updated 👍